k8s/manifests/deployment.yaml

```diff
@@ -17,14 +17,16 @@ spec:
         app: sentry-kubernetes
     spec:
       containers:
-        - image: tonyo/sentry-kubernetes:b7bf56753286880bb47a9caaa8621a3bf6c6b4d5
+        # - image: tonyo/sentry-kubernetes:b7bf56753286880bb47a9caaa8621a3bf6c6b4d5
+        - image: sentry-kubernetes:0.0.1
+          imagePullPolicy: IfNotPresent
           name: sentry-kubernetes
           env:
             - name: SENTRY_K8S_LOG_LEVEL
               value: debug
             - name: SENTRY_DSN
               # FIXME
-              value: ""
+              value: "https://c6f9a148ee0775891414b50b9af35959@o4506191942320128.ingest.sentry.io/4506191948087296"
```

Comment:
Let's not commit this DSN value, it won't be useful to anyone who doesn't have access to that org/project.